test/test.js

```diff
@@ -285,6 +285,112 @@ describe('justified-layout', function () {
 
 		});
 
+		it('should return widows with a left layout through the default', function () {
+
+			var geometry1LeftWidow = justifiedLayout([1, 1, 1, 1]);
+
+			expect(geometry1LeftWidow.boxes[0].left).toEqual(geometry1LeftWidow.boxes[3].left);
+
+		});
+
+		it('should return widows with a specified left layout', function () {
+
+			var geometry1LeftWidow = justifiedLayout([1, 1, 1, 1], {
+				widowLayoutStyle: 'left'
+			});
+
+			expect(geometry1LeftWidow.boxes[0].left).toEqual(geometry1LeftWidow.boxes[3].left);
+
+		});
+
+		function isThisWidowRowCentered(layout) {
```

Comment:
@jeremyruppel @alex-seville how do we feel about this function chilling in the middle of the tests? Prefer it at the top or inline where it's relevant? I have compelling arguments for both ways 😅